2024-07-01 20:34:23,335 INFO    StreamThr :18504 [internal.py:wandb_internal():85] W&B internal server running at pid: 18504, started at: 2024-07-01 20:34:23.334574
2024-07-01 20:34:23,337 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status
2024-07-01 20:34:23,350 INFO    WriterThread:18504 [datastore.py:open_for_write():87] open: /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/run-64baqblh.wandb
2024-07-01 20:34:23,350 DEBUG   SenderThread:18504 [sender.py:send():378] send: header
2024-07-01 20:34:23,380 DEBUG   SenderThread:18504 [sender.py:send():378] send: run
2024-07-01 20:34:23,772 INFO    SenderThread:18504 [dir_watcher.py:__init__():211] watching files in: /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/files
2024-07-01 20:34:23,772 INFO    SenderThread:18504 [sender.py:_start_run_threads():1123] run started: 64baqblh with start time 1719862463.334192
2024-07-01 20:34:23,782 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: check_version
2024-07-01 20:34:23,782 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: check_version
2024-07-01 20:34:23,991 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: run_start
2024-07-01 20:34:24,002 DEBUG   HandlerThread:18504 [system_info.py:__init__():26] System info init
2024-07-01 20:34:24,003 DEBUG   HandlerThread:18504 [system_info.py:__init__():41] System info init done
2024-07-01 20:34:24,003 INFO    HandlerThread:18504 [system_monitor.py:start():194] Starting system monitor
2024-07-01 20:34:24,003 INFO    SystemMonitor:18504 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-07-01 20:34:24,003 INFO    HandlerThread:18504 [system_monitor.py:probe():214] Collecting system info
2024-07-01 20:34:24,004 INFO    SystemMonitor:18504 [interfaces.py:start():188] Started cpu monitoring
2024-07-01 20:34:24,004 INFO    SystemMonitor:18504 [interfaces.py:start():188] Started disk monitoring
2024-07-01 20:34:24,004 INFO    SystemMonitor:18504 [interfaces.py:start():188] Started gpuapple monitoring
2024-07-01 20:34:24,013 INFO    SystemMonitor:18504 [interfaces.py:start():188] Started memory monitoring
2024-07-01 20:34:24,014 INFO    SystemMonitor:18504 [interfaces.py:start():188] Started network monitoring
2024-07-01 20:34:24,107 ERROR   HandlerThread:18504 [gpu_apple.py:probe():176] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 164, in probe
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:34:24,109 DEBUG   HandlerThread:18504 [system_info.py:probe():150] Probing system
2024-07-01 20:34:24,113 DEBUG   HandlerThread:18504 [system_info.py:_probe_git():135] Probing git
2024-07-01 20:34:24,131 DEBUG   HandlerThread:18504 [system_info.py:_probe_git():143] Probing git done
2024-07-01 20:34:24,131 DEBUG   HandlerThread:18504 [system_info.py:probe():198] Probing system done
2024-07-01 20:34:24,131 DEBUG   HandlerThread:18504 [system_monitor.py:probe():223] {'os': 'macOS-13.3-arm64-arm-64bit', 'python': '3.11.4', 'heartbeatAt': '2024-07-01T19:34:24.109557', 'startedAt': '2024-07-01T19:34:23.314736', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '/Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/hyperparameter_optimization.py', 'codePathLocal': 'hyperparameter_optimization.py', 'codePath': 'src/hyperparameter_optimization.py', 'git': {'remote': 'https://github.com/rajit906/Symmetry-invariant-Tractable-Generative-Models.git', 'commit': 'd49b859748630de0028c90a4d47f91818bf4e901'}, 'email': None, 'root': '/Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models', 'host': 'Rajits-MacBook-Air.local', 'username': 'rajit906', 'executable': '/Library/Frameworks/Python.framework/Versions/3.11/bin/python3', 'cpu_count': 8, 'cpu_count_logical': 8, 'disk': {'/': {'total': 228.27386474609375, 'used': 11.354015350341797}}, 'gpuapple': {'type': 'arm', 'vendor': 'Apple'}, 'memory': {'total': 8.0}}
2024-07-01 20:34:24,131 INFO    HandlerThread:18504 [system_monitor.py:probe():224] Finished collecting system info
2024-07-01 20:34:24,131 INFO    HandlerThread:18504 [system_monitor.py:probe():227] Publishing system info
2024-07-01 20:34:24,131 INFO    HandlerThread:18504 [system_monitor.py:probe():229] Finished publishing system info
2024-07-01 20:34:24,137 DEBUG   SenderThread:18504 [sender.py:send():378] send: files
2024-07-01 20:34:24,137 INFO    SenderThread:18504 [sender.py:_save_file():1389] saving file wandb-metadata.json with policy now
2024-07-01 20:34:24,175 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:34:24,231 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: python_packages
2024-07-01 20:34:24,232 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: python_packages
2024-07-01 20:34:24,232 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: stop_status
2024-07-01 20:34:24,232 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: stop_status
2024-07-01 20:34:24,476 DEBUG   SenderThread:18504 [sender.py:send():378] send: telemetry
2024-07-01 20:34:24,672 INFO    wandb-upload_0:18504 [upload_job.py:push():130] Uploaded file /var/folders/sx/y6c6vjtn04719m5bcb52hk3r0000gn/T/tmplrp8eacdwandb/kloadtut-wandb-metadata.json
2024-07-01 20:34:24,779 INFO    Thread-12 :18504 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/files/wandb-metadata.json
2024-07-01 20:34:24,779 INFO    Thread-12 :18504 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/files/requirements.txt
2024-07-01 20:34:26,292 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:34:28,401 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:34:28,501 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status_report
2024-07-01 20:34:30,513 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:34:32,623 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:34:33,529 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status_report
2024-07-01 20:34:34,753 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:34:36,859 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:34:38,557 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status_report
2024-07-01 20:34:38,990 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:34:39,250 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: stop_status
2024-07-01 20:34:39,250 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: stop_status
2024-07-01 20:34:41,117 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:34:43,243 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:34:44,472 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status_report
2024-07-01 20:34:45,358 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:34:47,491 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:34:49,493 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status_report
2024-07-01 20:34:49,630 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:34:51,760 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:34:53,879 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:34:54,259 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: stop_status
2024-07-01 20:34:54,260 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: stop_status
2024-07-01 20:34:54,618 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status_report
2024-07-01 20:34:54,931 INFO    Thread-12 :18504 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/files/config.yaml
2024-07-01 20:34:55,995 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:34:58,124 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:34:59,879 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status_report
2024-07-01 20:35:00,241 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:02,370 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:04,545 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:04,902 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status_report
2024-07-01 20:35:06,737 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:08,929 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:09,266 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: stop_status
2024-07-01 20:35:09,267 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: stop_status
2024-07-01 20:35:10,519 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status_report
2024-07-01 20:35:11,066 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:13,258 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:15,388 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:15,537 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status_report
2024-07-01 20:35:17,630 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:19,762 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:20,559 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status_report
2024-07-01 20:35:21,920 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:24,038 DEBUG   SystemMonitor:18504 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-07-01 20:35:24,040 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:24,041 DEBUG   SenderThread:18504 [sender.py:send():378] send: telemetry
2024-07-01 20:35:24,041 DEBUG   SenderThread:18504 [sender.py:send():378] send: stats
2024-07-01 20:35:24,271 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: stop_status
2024-07-01 20:35:24,271 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: stop_status
2024-07-01 20:35:25,674 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status_report
2024-07-01 20:35:26,057 INFO    Thread-12 :18504 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/files/config.yaml
2024-07-01 20:35:26,176 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:28,336 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:30,460 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:30,843 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status_report
2024-07-01 20:35:32,588 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:34,709 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:35,862 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status_report
2024-07-01 20:35:36,848 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:38,990 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:39,276 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: stop_status
2024-07-01 20:35:39,277 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: stop_status
2024-07-01 20:35:41,109 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:41,454 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status_report
2024-07-01 20:35:43,222 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:45,341 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:46,478 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status_report
2024-07-01 20:35:47,471 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:49,590 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:51,503 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status_report
2024-07-01 20:35:51,701 ERROR   gpuapple  :18504 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-07-01 20:35:53,235 DEBUG   SenderThread:18504 [sender.py:send():378] send: exit
2024-07-01 20:35:53,235 INFO    SenderThread:18504 [sender.py:send_exit():585] handling exit code: 0
2024-07-01 20:35:53,235 INFO    SenderThread:18504 [sender.py:send_exit():587] handling runtime: 89
2024-07-01 20:35:53,236 INFO    SenderThread:18504 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-07-01 20:35:53,237 INFO    SenderThread:18504 [sender.py:send_exit():593] send defer
2024-07-01 20:35:53,237 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: defer
2024-07-01 20:35:53,237 INFO    HandlerThread:18504 [handler.py:handle_request_defer():184] handle defer: 0
2024-07-01 20:35:53,237 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: defer
2024-07-01 20:35:53,237 INFO    SenderThread:18504 [sender.py:send_request_defer():609] handle sender defer: 0
2024-07-01 20:35:53,237 INFO    SenderThread:18504 [sender.py:transition_state():613] send defer: 1
2024-07-01 20:35:53,237 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: defer
2024-07-01 20:35:53,237 INFO    HandlerThread:18504 [handler.py:handle_request_defer():184] handle defer: 1
2024-07-01 20:35:53,237 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: defer
2024-07-01 20:35:53,237 INFO    SenderThread:18504 [sender.py:send_request_defer():609] handle sender defer: 1
2024-07-01 20:35:53,237 INFO    SenderThread:18504 [sender.py:transition_state():613] send defer: 2
2024-07-01 20:35:53,237 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: defer
2024-07-01 20:35:53,237 INFO    HandlerThread:18504 [handler.py:handle_request_defer():184] handle defer: 2
2024-07-01 20:35:53,237 INFO    HandlerThread:18504 [system_monitor.py:finish():203] Stopping system monitor
2024-07-01 20:35:53,238 INFO    HandlerThread:18504 [interfaces.py:finish():200] Joined cpu monitor
2024-07-01 20:35:53,238 DEBUG   SystemMonitor:18504 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-07-01 20:35:53,238 DEBUG   SystemMonitor:18504 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-07-01 20:35:53,238 INFO    HandlerThread:18504 [interfaces.py:finish():200] Joined disk monitor
2024-07-01 20:35:53,239 INFO    HandlerThread:18504 [interfaces.py:finish():200] Joined gpuapple monitor
2024-07-01 20:35:53,239 INFO    HandlerThread:18504 [interfaces.py:finish():200] Joined memory monitor
2024-07-01 20:35:53,239 INFO    HandlerThread:18504 [interfaces.py:finish():200] Joined network monitor
2024-07-01 20:35:53,239 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: defer
2024-07-01 20:35:53,239 INFO    SenderThread:18504 [sender.py:send_request_defer():609] handle sender defer: 2
2024-07-01 20:35:53,239 INFO    SenderThread:18504 [sender.py:transition_state():613] send defer: 3
2024-07-01 20:35:53,239 DEBUG   SenderThread:18504 [sender.py:send():378] send: stats
2024-07-01 20:35:53,240 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: defer
2024-07-01 20:35:53,240 INFO    HandlerThread:18504 [handler.py:handle_request_defer():184] handle defer: 3
2024-07-01 20:35:53,240 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: defer
2024-07-01 20:35:53,240 INFO    SenderThread:18504 [sender.py:send_request_defer():609] handle sender defer: 3
2024-07-01 20:35:53,240 INFO    SenderThread:18504 [sender.py:transition_state():613] send defer: 4
2024-07-01 20:35:53,240 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: defer
2024-07-01 20:35:53,240 INFO    HandlerThread:18504 [handler.py:handle_request_defer():184] handle defer: 4
2024-07-01 20:35:53,240 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: defer
2024-07-01 20:35:53,240 INFO    SenderThread:18504 [sender.py:send_request_defer():609] handle sender defer: 4
2024-07-01 20:35:53,240 INFO    SenderThread:18504 [sender.py:transition_state():613] send defer: 5
2024-07-01 20:35:53,240 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: defer
2024-07-01 20:35:53,240 INFO    HandlerThread:18504 [handler.py:handle_request_defer():184] handle defer: 5
2024-07-01 20:35:53,240 DEBUG   SenderThread:18504 [sender.py:send():378] send: summary
2024-07-01 20:35:53,240 INFO    SenderThread:18504 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-07-01 20:35:53,240 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: defer
2024-07-01 20:35:53,240 INFO    SenderThread:18504 [sender.py:send_request_defer():609] handle sender defer: 5
2024-07-01 20:35:53,240 INFO    SenderThread:18504 [sender.py:transition_state():613] send defer: 6
2024-07-01 20:35:53,240 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: defer
2024-07-01 20:35:53,240 INFO    HandlerThread:18504 [handler.py:handle_request_defer():184] handle defer: 6
2024-07-01 20:35:53,241 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: defer
2024-07-01 20:35:53,241 INFO    SenderThread:18504 [sender.py:send_request_defer():609] handle sender defer: 6
2024-07-01 20:35:53,241 INFO    SenderThread:18504 [sender.py:transition_state():613] send defer: 7
2024-07-01 20:35:53,241 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: status_report
2024-07-01 20:35:53,241 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: defer
2024-07-01 20:35:53,241 INFO    HandlerThread:18504 [handler.py:handle_request_defer():184] handle defer: 7
2024-07-01 20:35:53,241 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: defer
2024-07-01 20:35:53,241 INFO    SenderThread:18504 [sender.py:send_request_defer():609] handle sender defer: 7
2024-07-01 20:35:54,186 INFO    Thread-12 :18504 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/files/wandb-summary.json
2024-07-01 20:35:54,186 INFO    Thread-12 :18504 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/files/output.log
2024-07-01 20:35:54,240 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: poll_exit
2024-07-01 20:35:55,237 INFO    SenderThread:18504 [sender.py:transition_state():613] send defer: 8
2024-07-01 20:35:55,237 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: poll_exit
2024-07-01 20:35:55,238 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: defer
2024-07-01 20:35:55,238 INFO    HandlerThread:18504 [handler.py:handle_request_defer():184] handle defer: 8
2024-07-01 20:35:55,238 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: defer
2024-07-01 20:35:55,238 INFO    SenderThread:18504 [sender.py:send_request_defer():609] handle sender defer: 8
2024-07-01 20:35:55,238 INFO    SenderThread:18504 [sender.py:transition_state():613] send defer: 9
2024-07-01 20:35:55,238 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: defer
2024-07-01 20:35:55,238 INFO    HandlerThread:18504 [handler.py:handle_request_defer():184] handle defer: 9
2024-07-01 20:35:55,238 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: defer
2024-07-01 20:35:55,238 INFO    SenderThread:18504 [sender.py:send_request_defer():609] handle sender defer: 9
2024-07-01 20:35:55,239 INFO    SenderThread:18504 [dir_watcher.py:finish():358] shutting down directory watcher
2024-07-01 20:35:55,245 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: poll_exit
2024-07-01 20:35:56,196 INFO    SenderThread:18504 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/files/output.log
2024-07-01 20:35:56,196 INFO    SenderThread:18504 [dir_watcher.py:finish():388] scan: /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/files
2024-07-01 20:35:56,196 INFO    SenderThread:18504 [dir_watcher.py:finish():402] scan save: /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/files/requirements.txt requirements.txt
2024-07-01 20:35:56,196 INFO    SenderThread:18504 [dir_watcher.py:finish():402] scan save: /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/files/output.log output.log
2024-07-01 20:35:56,199 INFO    SenderThread:18504 [dir_watcher.py:finish():402] scan save: /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/files/config.yaml config.yaml
2024-07-01 20:35:56,199 INFO    SenderThread:18504 [dir_watcher.py:finish():402] scan save: /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/files/wandb-summary.json wandb-summary.json
2024-07-01 20:35:56,202 INFO    SenderThread:18504 [dir_watcher.py:finish():402] scan save: /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/files/wandb-metadata.json wandb-metadata.json
2024-07-01 20:35:56,202 INFO    SenderThread:18504 [sender.py:transition_state():613] send defer: 10
2024-07-01 20:35:56,203 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: poll_exit
2024-07-01 20:35:56,203 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: defer
2024-07-01 20:35:56,205 INFO    HandlerThread:18504 [handler.py:handle_request_defer():184] handle defer: 10
2024-07-01 20:35:56,206 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: defer
2024-07-01 20:35:56,206 INFO    SenderThread:18504 [sender.py:send_request_defer():609] handle sender defer: 10
2024-07-01 20:35:56,206 INFO    SenderThread:18504 [file_pusher.py:finish():169] shutting down file pusher
2024-07-01 20:35:56,247 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: poll_exit
2024-07-01 20:35:56,248 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: poll_exit
2024-07-01 20:35:56,626 INFO    wandb-upload_0:18504 [upload_job.py:push():130] Uploaded file /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/files/requirements.txt
2024-07-01 20:35:56,645 INFO    wandb-upload_1:18504 [upload_job.py:push():130] Uploaded file /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/files/output.log
2024-07-01 20:35:56,672 INFO    wandb-upload_3:18504 [upload_job.py:push():130] Uploaded file /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/files/wandb-summary.json
2024-07-01 20:35:56,698 INFO    wandb-upload_2:18504 [upload_job.py:push():130] Uploaded file /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/files/config.yaml
2024-07-01 20:35:56,903 INFO    Thread-11 (_thread_body):18504 [sender.py:transition_state():613] send defer: 11
2024-07-01 20:35:56,905 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: defer
2024-07-01 20:35:56,905 INFO    HandlerThread:18504 [handler.py:handle_request_defer():184] handle defer: 11
2024-07-01 20:35:56,905 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: defer
2024-07-01 20:35:56,905 INFO    SenderThread:18504 [sender.py:send_request_defer():609] handle sender defer: 11
2024-07-01 20:35:56,906 INFO    SenderThread:18504 [file_pusher.py:join():175] waiting for file pusher
2024-07-01 20:35:56,907 INFO    SenderThread:18504 [sender.py:transition_state():613] send defer: 12
2024-07-01 20:35:56,909 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: defer
2024-07-01 20:35:56,909 INFO    HandlerThread:18504 [handler.py:handle_request_defer():184] handle defer: 12
2024-07-01 20:35:56,909 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: defer
2024-07-01 20:35:56,910 INFO    SenderThread:18504 [sender.py:send_request_defer():609] handle sender defer: 12
2024-07-01 20:35:56,910 INFO    SenderThread:18504 [file_stream.py:finish():601] file stream finish called
2024-07-01 20:35:57,253 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: poll_exit
2024-07-01 20:35:57,364 INFO    SenderThread:18504 [file_stream.py:finish():605] file stream finish is done
2024-07-01 20:35:57,364 INFO    SenderThread:18504 [sender.py:transition_state():613] send defer: 13
2024-07-01 20:35:57,364 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: poll_exit
2024-07-01 20:35:57,364 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: defer
2024-07-01 20:35:57,365 INFO    HandlerThread:18504 [handler.py:handle_request_defer():184] handle defer: 13
2024-07-01 20:35:57,365 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: defer
2024-07-01 20:35:57,365 INFO    SenderThread:18504 [sender.py:send_request_defer():609] handle sender defer: 13
2024-07-01 20:35:57,365 INFO    SenderThread:18504 [sender.py:transition_state():613] send defer: 14
2024-07-01 20:35:57,365 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: defer
2024-07-01 20:35:57,365 INFO    HandlerThread:18504 [handler.py:handle_request_defer():184] handle defer: 14
2024-07-01 20:35:57,365 DEBUG   SenderThread:18504 [sender.py:send():378] send: final
2024-07-01 20:35:57,365 DEBUG   SenderThread:18504 [sender.py:send():378] send: footer
2024-07-01 20:35:57,365 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: defer
2024-07-01 20:35:57,365 INFO    SenderThread:18504 [sender.py:send_request_defer():609] handle sender defer: 14
2024-07-01 20:35:57,365 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: poll_exit
2024-07-01 20:35:57,366 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: poll_exit
2024-07-01 20:35:57,366 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: poll_exit
2024-07-01 20:35:57,366 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: server_info
2024-07-01 20:35:57,366 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: poll_exit
2024-07-01 20:35:57,366 DEBUG   SenderThread:18504 [sender.py:send_request():405] send_request: server_info
2024-07-01 20:35:57,367 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: get_summary
2024-07-01 20:35:57,367 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: sampled_history
2024-07-01 20:35:57,367 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: internal_messages
2024-07-01 20:35:57,501 INFO    MainThread:18504 [wandb_run.py:_footer_history_summary_info():3994] rendering history
2024-07-01 20:35:57,501 INFO    MainThread:18504 [wandb_run.py:_footer_history_summary_info():4026] rendering summary
2024-07-01 20:35:57,501 INFO    MainThread:18504 [wandb_run.py:_footer_sync_info():3953] logging synced files
2024-07-01 20:35:57,501 DEBUG   HandlerThread:18504 [handler.py:handle_request():158] handle_request: shutdown
2024-07-01 20:35:57,502 INFO    HandlerThread:18504 [handler.py:finish():882] shutting down handler
2024-07-01 20:35:58,372 INFO    WriterThread:18504 [datastore.py:close():296] close: /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240701_203423-64baqblh/run-64baqblh.wandb
2024-07-01 20:35:58,504 INFO    SenderThread:18504 [sender.py:finish():1545] shutting down sender
2024-07-01 20:35:58,504 INFO    SenderThread:18504 [file_pusher.py:finish():169] shutting down file pusher
2024-07-01 20:35:58,504 INFO    SenderThread:18504 [file_pusher.py:join():175] waiting for file pusher
